docs/faq.md

```diff
@@ -63,4 +63,68 @@ The gene expression data files available for download report all possible genes
+
+## What if I want to use Seurat instead of Bioconductor? 
+
+The files available for download contain [`SingleCellExperiment` objects](http://bioconductor.org/books/3.13/OSCA.intro/the-singlecellexperiment-class.html). 
+If desired, these can be converted into Seurat objects. 
+
+You will need to [install and load the `Seurat` package](https://satijalab.org/seurat/articles/install.html) to work with Seurat objects.
+
+For libraries that only contain RNA-sequencing data (i.e. do not have a CITE-seq library found in the `altExp` of the `SingleCellExperiment` object), you can use the following commands:
+
+```
+library(Seurat)
+library(SingleCellExperiment)
+
+# read in RDS file 
+sce <- readRDS("SCPCL000000_filtered.rds")
+
+# grab counts matrix from SingleCellExperiment needed to build the Seurat Object
+counts <- counts(sce)
+
+# colData and rowData will need to be converted to a data.frame to be used in the Seurat object
+cell_metadata <- as.data.frame(colData(sce))
+row_metadata <- as.data.frame(rowData(sce))
+
+# create seurat object 
+seurat_object <- CreateSeuratObject(counts = counts, 
+                                    assay = "RNA",
+                                    project = "SCPCL000000")
+```
+The above code will only maintain information found in the original counts matrix from the `SingleCellExperiment`.
+Optionally, if you would like to keep the included cell and gene associated metadata during conversion to the Seurat object you can perform the below additional steps: 
+
+```
+# colData and rowData will need to be converted to a data.frame to be used in the Seurat object
+cell_metadata <- as.data.frame(colData(sce))
+row_metadata <- as.data.frame(rowData(sce))
+
+# add cell metadata (colData) from SingleCellExperiment to Seurat
+seurat_object@meta.data <- cell_metadata
+
+# add row metadata (rowData) from SingleCellExperiment to Seurat 
+seurat_object[["RNA"]]@meta.features <- row_metadata
+
+# add metadata from SingleCellExperiment to Seurat
+seurat_object@misc <- metadata(sce)
+```
+
+In order to convert `SingleCellExperiment` objects from libraries that have counts from both RNA-sequencing and CITE-seq, you can use the following additional commands.
+This adds a second assay containing the CITE-seq counts and associated feature data to an existing Seurat object:
+
+```
+# create assay object in Seurat from CITE-seq counts found in altExp(SingleCellExperiment)
+cite_counts <- counts(altExp(sce))
+cite_assay <- CreateAssayObject(counts = cite_counts)
+
+# optional: convert rowData to data.frame for use in Seurat object
+cite_row_metadata <- as.data.frame(rowData(altExp(sce)))
+
+# optional: add row metadata (rowData) from altExp to assay 
+cite_assay@meta.features = cite_row_metadata
```

Comment:
I think we can combine these two with one comment (also changing assignment operator)
```suggestion
# optional: add row metadata (rowData) from altExp to assay 
cite_row_metadata <- as.data.frame(rowData(altExp(sce)))
cite_assay@meta.features <- cite_row_metadata
```